src/components/override-sdk/infra/View/View.tsx

```diff
@@ -35,8 +35,8 @@ export default function View(props) {
   useEffect(() => {
     PCore.getPubSubUtils().subscribe('languageToggleTriggered', locale => {
       if (!locale.localeRef.includes(props.localeReference)) {
-        locale.localeRef.push(props.localeReference);
-        PCore.getLocaleUtils().loadLocaleResources([props.localeReference]);
+        // locale.localeRef.push(props.localeReference);
```

Comment:
This section of code 38 - 39 is commented out, is this needed if not can we take it out?